lms/templates/dashboard.html

```diff
@@ -194,19 +194,20 @@
                 show_courseware_link = show_courseware_links_for.get(session_id, False)
                 cert_status = cert_statuses.get(session_id)
                 can_refund_entitlement = entitlement and entitlement.is_entitlement_refundable()
+                partner_managed_enrollment = enrollment.mode == 'masters'
```

Comment:
Why is this necessary?

Reply:
@shaidar this is need as an argument for passing to `<%include file='dashboard/_dashboard_course_listing.html' %>`

Reply:
If I remove that argument, I get that error
![image](https://user-images.githubusercontent.com/4043989/97318531-c89b2c00-188d-11eb-84d0-94b8afab517f.png)

Reply:
So I have copied it from the edx-platform default dashboard theme.

Reply:
Of course you're get that unless u remove it from the include statement below. I tried loading the dashboard by just adding the `if` conditions that you've added below, and that seems to solve the problem. That's why I was wondering whether the `partner_managed_enrollment` is in fact needed or not. 
Can we remove it along with its inclusion below?

Reply:
It does not workout for me that's why I have added it and I have provisioned on the upstream master.

Reply:
You removed it from line 197 and from line 210? Cause I just made the change on xPRO RC without adding any reference to `partner_managed_enrollment` and it seems to be working fine

Reply:
ok will remove it.

Reply:
I am removing it on @shaidar suggestion but I am getting error on `upstream master` for that. As I am provisioned on `upstream` so have no idea about the `juniper` release.